backend/app/assets/javascripts/spree/backend/turbolinks-configuration.js.erb

```diff
@@ -0,0 +1,20 @@
+// comment line necessary for correct interpolation of use_turbolinks
+Turbolinks.supported = <%= Spree::Backend::Config.use_turbolinks %>;
```

Comment:
Hi @tvdeyen , sorry to pester .. just whenever you get to it .. if you could rebase your branch on top of master now that that preparation PR has been merged .. as otherwise I think I won't be able to get the tests to green?